Neox Scanlator/presence.ts

```diff
@@ -0,0 +1,112 @@
+var presence = new Presence({
+    clientId: "704585837949747330"
+});
+
+presence.on("UpdateData", async () => {
+    const presenceData: presenceData = {
+        largeImageKey: "logo",
+        startTimestamp: Math.floor(Date.now() / 1000)
+    };
+    const path: any = document.location.pathname;
+    let PesquisaTexto: any;
+    let UsuarioTexto: any;
+    let OrdenarTexto: any;
+    let OrdenarTextoObra: any;
+    let GeneroTexto: any;
+    let StatusContaTexto: any;
+    let BlogText: any;
+    let opcaoLeitor: any;
+    let tipoObra: any;
+    let generoObra: any;
+    let spanObra: any;
+    let nomeObra: any;
+    let nomeObraLeitor: any;
+    let capituloLeitor: any;
+    let seasonLeitor: any;
+    let paginaLeitor: any;
+    let postagemData: any;

```

Comment:
use commas:
Let PesquisaTexto, UsuarioTexto, etc.
They will all be any automatically that way.